pylgbst/hub.py

```diff
@@ -287,3 +294,126 @@ def __init__(self, connection=None):
         if connection is None:
             connection = get_connection_auto(hub_name=self.DEFAULT_NAME)
         super(TrainHub, self).__init__(connection)
+
+class ControlPlusHub(Hub):
+    """
+    Class implementing Lego Control + Hub
+    :type motor_A: EncodedMotor
+    :type motor_B: EncodedMotor
+    :type motor_C: EncodedMotor
+    :type motor_D: EncodedMotor
+    :type led: LEDRGB
+    :type current: Current
+    :type voltage: Voltage
+    :type button: Button
+    :type tilt_sensor: TiltSensor  
+    :type accelerometer: Accelerometer
+    :type unknown: Periphiral, that's used for unknown ports  
+    """
+    DEFAULT_NAME = "LEGO Control + Hub"
+
+    # PORTS
+    PORT_A = 0x00
+    PORT_B = 0x01
+    PORT_C = 0x02
+    PORT_D = 0x03
+    PORT_LED = 0x32
+    PORT_CURRENT = 0x3B
+    PORT_VOLTAGE = 0x3C
+    PORT_TILT_SENSOR = 0x63
+    PORT_ACCELEROMETER_A = 0x62
+    PORT_ACCELEROMETER_B = 0x61
+    PORT_UNKNOWN_A = 0x3d 
```

Comment:
These are not unknown. Look into https://github.com/sharpbrick/powered-up/blob/master/src/SharpBrick.PoweredUp/Hubs/TechnicMediumHub.cs

I am also not aware of multiple accelerometer (one of the is a gyrosensor). The unknown ones are temperature and gesture. Temperature is easy, gesture ... i have not found any material on them.